docs/getting_started.txt

```diff
@@ -1,16 +1,55 @@
 ==========================
-Automating an application
+Getting Started Guide
 ==========================
 
 Once you have installed pywinauto - how do you get going?
+The very first necessary thing is to determine which accessibility 
+technology (pywinauto's backend) could be used for your application.
 
-Sit back and have a look at a little movie
--------------------------------------------
+The list of supported accessibility technologies on Windows:
 
-Jeff Winkler has created a nice screencast of using pywinauto, you
-can see it at :
+ * Win32 API (``backend="win32"``)
+    - MFC, VB6, VCL, simple WinForms controls and most of the old legacy apps
 
-   http://showmedo.com/videos/video?name=UsingpyWinAutoToControlAWindowsApplication&fromSeriesID=7
+ * MS UI Automation (``backend="uia"``)
+    - WinForms, WPF, Store apps, Qt, browsers (Chrome requires 
+    ``--force-renderer-accessibility`` cmd flag before running)
+
+Note that custom properties and controls are not supported because of 
+comtypes Python library restrictions.
+
+Not supported: Java AWT/Swing, GTK+, Tkinter.
+
+AT SPI on Linux and Apple Accessibility API are in the long term plans so far.
+
+
+GUI Objects Inspection
+--------------------
+If you're still not sure which backend is most appropriate for you, 
+then try using object inspection tools that are free available.
+
+ * **Spy++** is included into MS Visual Studio distribution and is available in Start
+   menu. It uses Win32 API. It means if Spy++ can show all the controls the ``"win32"``
+   backend is what you need. **AutoIt Window Info** tool is a kind of Spy++ clone.
+
+ * **Inspect.exe** is another great tool created by Microsoft. It's included into
+   Windows SDK so that can be found in the following location on x64 Windows:
+
+ ::
+
+    C:\Program Files (x86)\Windows Kits\<winver>\bin\x64
+
+Inspect.exe is the most useful in **UIA mode** (using MS UI Automation). If it 
+can show more controls and their properties than Spy++, probably the ``"uia"`` 
```

Comment:
The most useful mode of Inspect.exe utility is **UIA mode** (using MS UI Automation) . If it shows more controls and their properties than Spy++, probably, the ``"uia"``